From 774bbab7184b43a549d4cb7407a5f38fb1061476 Mon Sep 17 00:00:00 2001
From: Jacob Stiffler <j-stiffler@ti.com>
Date: Tue, 3 Nov 2015 07:26:03 -0500
Subject: [PATCH] Platform.xdc: Consolidate DDR3 memory

* Reduce DDR3 footprint from 9 MB to 5 MB to fit in upstream
  remoteproc dsp1 CMA region

Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 monitor_vayu/custom_rsc_table_vayu_dsp.h      | 4 ++--
 monitor_vayu/ocl_platforms/am57x/Platform.xdc | 8 ++++----
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/monitor_vayu/custom_rsc_table_vayu_dsp.h b/monitor_vayu/custom_rsc_table_vayu_dsp.h
index 85c0c33..f89825e 100644
--- a/monitor_vayu/custom_rsc_table_vayu_dsp.h
+++ b/monitor_vayu/custom_rsc_table_vayu_dsp.h
@@ -88,8 +88,8 @@
 #define DSP_TILER_MODE_3        0x78000000
 
 // Static DDR range used by monitor (DDR3 region in Platform.xdc)
-#define DSP_MEM_DDR             0xFE800000
-#define DSP_MEM_DDR_SIZE        (SZ_1M * (8 + 1))
+#define DSP_MEM_DDR             0xFEC00000
+#define DSP_MEM_DDR_SIZE        (SZ_1M * (4 + 1))
 
 // CMEM buffers mapped by MMU to PHYS_MEM_IOBUFS
 #define DSP_MEM_IOBUFS          0x80000000
diff --git a/monitor_vayu/ocl_platforms/am57x/Platform.xdc b/monitor_vayu/ocl_platforms/am57x/Platform.xdc
index cd0ce2f..1638743 100644
--- a/monitor_vayu/ocl_platforms/am57x/Platform.xdc
+++ b/monitor_vayu/ocl_platforms/am57x/Platform.xdc
@@ -38,19 +38,19 @@ config ti.platforms.generic.Platform.Instance CPU =
       * DDR Related regions
       *----------------------------------------------------------------------*/
       [ "DDR3",         { name: "DDR3", 
-                          base: 0xFE800000, 
-			  len:  0x00400000,
+                          base: 0xFEC00000, 
+			  len:  0x003E0000,
                           space: "code/data", 
 			  access: "RWX", } ],
 
       /* Each DSP core uses its own framework components data structure */
       [ "DDR3_FC0",     { name: "DDR3_FC0",
-                          base: 0xFEC00000,
+                          base: 0xFEFE0000,
 			  len:  0x00010000,
                           space: "code/data",
 			  access: "RWX", } ],
       [ "DDR3_FC1",     { name: "DDR3_FC1",
-                          base: 0xFEC10000,
+                          base: 0xFEFF0000,
 			  len:  0x00010000,
                           space: "code/data",
 			  access: "RWX", } ],
-- 
1.9.1

